CHANGELOG.rst

```diff
@@ -1,14 +1,22 @@
 Changelog for Xem Wrapper
 =========================
 
+
+0.3.0 (2021-03-11)
```

Comment:
do you think we need to release a new version right now? or just say "unreleased"?